UMutils.py

```diff
@@ -246,24 +246,17 @@ def logistic(x, k, offset):
         print("%s fell through the cracks" % (user_id))
         sys.exit()
 
-    if user_id not in ucs["uuid"].values:  # if this is the user's first task
-        cur_ucs = 0.5
-        var_scores = np.var(task_scores.iloc[-n:])
-        c = logistic(var_scores / (np.log(num_task_scores + 1) / (np.log(a))), 10, 0.2)
-        new_ucs = cur_ucs * (1 - c) + (c) * last_task_score
-        query = "INSERT INTO `ucs` (`uuid`, `score`) VALUES (%s, %s)"
-        cursor.execute(query, (user_id, new_ucs))
-        client.connection.commit()
-    else:
+    cur_ucs = -1
```

Comment:
This is cleaning up duplicate code and moving more into the SQL Handler so the pandas replacement could work better